tests/Unit/PointwiseFunctions/GeneralRelativity/Test_ComputeGhQuantities.cpp

```diff
@@ -574,4 +586,84 @@ SPECTRE_TEST_CASE("Unit.PointwiseFunctions.GeneralRelativity.GhQuantities",
                                     upper_bound);
   test_shift_deriv_functions_analytic(solution, grid_size, lower_bound,
                                       upper_bound);
+
+  // Check that compute items work correctly in the DataBox
+  // First, check that the names are correct
+  CHECK(GeneralizedHarmonic::Tags::DerivSpatialMetricCompute<
+            3, Frame::Inertial>::name() == "deriv(SpatialMetric)");
+  CHECK(GeneralizedHarmonic::Tags::DerivLapseCompute<3,
+                                                     Frame::Inertial>::name() ==
+        "deriv(Lapse)");
+  CHECK(GeneralizedHarmonic::Tags::DerivShiftCompute<3,
+                                                     Frame::Inertial>::name() ==
+        "deriv(Shift)");
+
+  // Check that the compute items return the correct values
+  MAKE_GENERATOR(generator);
+  std::uniform_real_distribution<> dist(-1., 1.);
+  std::uniform_real_distribution<> dist_positive(1., 2.);
+  const auto nn_generator = make_not_null(&generator);
```

Comment:
I have a mild philosophical preference for not using these. They are reminiscent of the type-prefix notation that we don't wish to use in the code base, and slightly undermine what I understand to be one of the motivations of passing by `gsl::not_null`, which is to indicate in any call the quantities that could be altered during the function execution. 
I admit this particular case doesn't have much pathology due to how dists and generators are used in the implementation, but also tests should be examples of 'good methodology'.